docs/tutorials/mastering.md

```diff
@@ -174,25 +194,44 @@ Create VMs with Multipass:
 
 ```console
 multipass launch -c 1 -m 1G -d 10G -n dns --network mgmt-br jammy
+```
+
+```console
```

Comment:
Why do we split the blocks in individual commands? This is making the tutorial more laborious to follow, because each command needs to be copy-pasted individually.

Reply:
It definitely becomes tedious later on when installing and configuration microk8s for each of the VMs...

Reply:
OK, I reverted it ;)